### DISCUSSION TITLE
Welcome to zigpy Discussions!

### DISCUSSION THREAD

**Discussion by Adminiuga** [zigpy/zigpy#570] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by MattWestb** [comment]
I think this forum is great for no real technical issues in our systems and user question that can helping users getting there systems working good.
Also for futures requests and encatchment is better putting here then issues in the issue tracing part of this git.

For future request should we putting them together on one new thread and linking already posted issues in it so getting all together and also getting new ideas ther for brainstorming and in/out put to/from our devs ?

And in the long run the new wiki can helping user with problems that dont need devs attention but saving time and tear for us. Im not native english speaking / writing and my gramma is very bad so im not gonna spamming the wiki that i letting for other that having that gift ;-)

**Reply by MattWestb** [reply]
By the way @Hedda the Christmas man was here for nearly one week ago and have opening one forum for brainstorming and we is missing you my old X fellow countryman !!
I cant being alone spamming the forum here  ;-(

**Reply by Hedda** [reply]
> Im not native english speaking / writing and my gramma is very bad so im not gonna spamming the wiki

@MattWestb You really need to install Grammarly free AI-powered writing assistant for Chrome  -> https://www.grammarly.com/

I'm not a native English speaker either but I use Grammarly which help me with spellchecking and grammar a lot 👍 👍

**Reply by MattWestb** [reply]
I have nearly killed one college by writing "påka" (Eng "Hitting someone in the head with somthing heavy for killing the person" as the vikings was doing some times ago) instead of "Påkalla"  (Eng "getting attention in the quest") and she was asking witch person I should killing, thanks to the grammar correction is MS Word.

**Comment by MattWestb** [comment]
Is it possible doing 2 new categories for for discussions so not getting the general category so crowded ❔
One for software ideas 💡 📃 and one for hardware ideas 💡🛠️/⚒️.
If you like have all in generalenetal  its OK for My but we are some pretty good spammers around here ;-)

And some [GITS](https://gist.github.com/rxaviers/7360908) if you like using.
